perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -1544,76 +1544,76 @@ public List<AttributeDefinition> getAttributesDefinitionWithRights(PerunSession
 			AttributeDefinition attrDef = iterator.next();
 
 			if (this.isFromNamespace(sess, attrDef, NS_USER_FACILITY_ATTR) && user != null && facility != null) {
-				if (!AuthzResolver.isAuthorizedForAttribute(sess, READ, attrDef, user, facility)) {
+				if (!AuthzResolver.isAuthorizedForAttribute(sess, READ, attrDef, user, facility, true)) {
 					iterator.remove();
 				} else {
-					attrDef.setWritable(AuthzResolver.isAuthorizedForAttribute(sess, WRITE, attrDef, user, facility));
+					attrDef.setWritable(AuthzResolver.isAuthorizedForAttribute(sess, WRITE, attrDef, user, facility, false));
```

Comment:
Wouldn't the true/false be in the opposite order (`false` for `READ` and `true` for `WRITE`)? We want to check MFA for `WRITE`, am I right?

Reply:
NOTE: I agree that this order is correct after discussion